app/services/shopify_import/products/create.rb

```diff
@@ -30,6 +31,31 @@ def add_tags
         @spree_product.save!
       end
 
+      def add_option_types
+        return if shopify_product.options.blank?
+
+        @spree_product.update!(option_type_ids: create_option_types)
+      end
+
+      def create_option_types
+        option_types_data.map do |option_type, option_values|
+          spree_option_type = Spree::OptionType.where(name: option_type).first_or_create!(presentation: option_type)
+          create_option_values(spree_option_type, option_values)
+          spree_option_type.id
+        end
+      end
+
+      def create_option_values(spree_option_type, option_values)
+        option_values.each do |option_value|
+          spree_option_type
+            .option_values.where('lower(name) = ?', option_value)
+            .first_or_create! do |ov|
```

Comment:
Seems like `first_or_create` has been "silently deprecated". You should use `find_or_create_by` instead:
https://github.com/rails/rails/commit/0096f53b25e68c3fc79429253f816fff4a4ee596